src/CollectBlock.ts

```diff
@@ -61,21 +61,21 @@ const equipToolOptions = {
 }
 
 async function mineBlock (bot: Bot, block: Block, options: CollectOptionsFull): Promise<void> {
-  if (bot.blockAt(block.position) !== block) {
+  // @ts-expect-error
+  if (bot.blockAt(block.position)?.type !== block.type || bot.blockAt(block.position)?.type === 0 || !bot.pathfinder.movements.safeToBreak(block)) {
```

Comment:
There are other types off air too like `void_air` and `cave_air`. So checking against 0 might not work in some edge cases.